### PR TITLE
Fix benchmarks of pallet-lending

### PR DESCRIPTION
**Pull request by andor0**
## Issue
https://app.clickup.com/t/2azjn0b

## Description
Also I added benchmarks for `liquidate` extrinsic, also I updated tests.

### CODE REVIEW

File: frame/lending/src/lib.rs [hunk 1]
```diff
@@ -419,6 +422,8 @@ pub mod pallet {
 		CannotRepayMoreThanTotalDebt,
 
 		BorrowRentDoesNotExist,
+
+		MaxLiquidationBatchSizeExceed,
```
Comment (benluelo) [inline frame/lending/src/lib.rs:426]:
```suggestion
		MaxLiquidationBatchSizeExceeded,
```

**Review by benluelo — COMMENTED** [review]
LGTM, just a small nit on spelling

File: frame/lending/src/lib.rs (hunk 1 above)
Comment (andor0) [reply frame/lending/src/lib.rs]: Fixed.

**Review by benluelo — APPROVED** [review]

File: frame/lending/src/lib.rs [hunk 2]
```diff
@@ -749,14 +754,18 @@ pub mod pallet {
 		/// liquidation.
 		/// - `origin` : Sender of this extrinsic.
 		/// - `market_id` : Market index from which `borrower` has taken borrow.
-		#[pallet::weight(<T as Config>::WeightInfo::liquidate(borrowers.len() as Weight))]
+		#[pallet::weight(<T as Config>::WeightInfo::liquidate(borrowers.len() as u32))]
```
Comment (dzmitry-lahoda) [inline frame/lending/src/lib.rs:757]: not sure. may be we should use usize?

**Review by dzmitry-lahoda — APPROVED** [review]

File: frame/lending/src/lib.rs (hunk 2 above)
Comment (andor0) [reply frame/lending/src/lib.rs:757]: a auto-created file with weights (`runtime/dali/src/weights/lending.rs`) will have `fn liquidate(b: u32, ) -> Weight { ... }` independ on what we will use `u32` or `usize`. i.e. we will get a broken build with `usize`.

**Pull request merged**